libs/platform/src/lib/components/upload-collection/dialogs/move-to/move-to.component.ts

```diff
@@ -0,0 +1,125 @@
+import { Component, Inject, OnInit } from '@angular/core';
+
+import { DIALOG_REF, DialogRef, DialogService, DialogConfig } from '@fundamental-ngx/core';
+import { take } from 'rxjs/operators';
+
+import { UploadCollectionItem, UploadCollectionFolder } from '../../models/upload-collection.models';
+import { NewFolderComponent } from '../new-folder';
+
+function isObject<T>(item: T): boolean {
+    return typeof item === 'object' && !Array.isArray(item) && item !== null;
+}
+
+@Component({
+    templateUrl: './move-to.component.html',
+    styles: [
+        `
+            .fd-list__item--title-background {
+                background: #f2f2f2;
```

Comment:
This is not themable. If you change the theme, the hard-coded color will stay the same.